crm_request/models/request.py

```diff
@@ -139,6 +153,10 @@ def message_new(self, msg, custom_values=None):
             }, options)
             if partner:
                 defaults['partner_id'] = partner.id
+                defaults['language'] = partner.lang
+
+        if 'language' not in defaults:
+            defaults['language'] = self.detect_lang(defaults['description'])
```

Comment:
The key description does not exist anymore here. You can do that after the super:
`request = self.browse(id)`
`request.language = self.detect_lang(request.description)`
@ecino what do you think?

Reply:
Yes, it looks good. Note that `super` is returning the `id` of the request.